Documentation.md

```diff
@@ -105,8 +105,83 @@ Users have to fill in the fields during the installation process first, on the i
 
 ![Authentication admin](./assets/images/marketplace-configure.png)
 
-## App Settings
-This part is still under development. This is where you will be able to add specific configuration steps such as mapping.
+## App Settings (in progress)
+As the authentication section, you can configure setting parameters that users will need to configure.
+
+**Define app's setting fields**
+You can add fields and define the type of entry you need.
+
+* Label: it’s the name of your field, this is what users will see.
+* Type: it defines the type of entry.
+* ID: you should choose the name of your field ID according to your code.
+
+You can add as much fields as you need.
+This is the second steps of users installation on the iAdvize Marketplace.
+Fields appear to users according to their order of creation (the 1st entry created is the 1st on displayed on the page).
+
+* Configure your [verification Url](#verification-url)
+This is the URL on which we will check your app. We'll call this URL and your connector will be sent back if the setting parameters have been correctly set up to go to the next installation step.
+
+
+Users have to fill in the fields during the installation process second, on the iAdvize Marketplace.
+
+
+## Verification url
+
+In [app authentication](#app-authentication) and [app-settings](#app-settings), you can configure verification url.
+This is the URL on which we will check the authentication and setting parameters of your new users during their connector's installation.
+We'll call this URL and your connector will be sent back if the authentication or setting parameters have been correctly set up to go to the next installation step.
+
+### Request payload
+
+This is the data passed to your verification url by payload body of a POST Request.
```

Comment:
Here is the information sent to your verification url as a payload body of a POST Request.